config.py

```diff
@@ -1969,3 +2018,420 @@ def annotations_to_events(
+                msg = f'    {eog_ch_name} -> eog'
+                logger.info(gen_log_message(message=msg, step=0,
+                                            subject=subject,
+                                            session=session))
+                raw.set_channel_types({eog_ch_name: 'eog'})
+
+
+def set_eeg_montage(raw, subject, session) -> None:
+    """Set an EEG template montage if requested.
+
+    Modifies ``raw`` in-place.
+    """
+    montage_name = eeg_template_montage
+    if get_datatype() == 'eeg' and montage_name:
+        msg = (f'Setting EEG channel locations to template montage: '
+               f'{montage_name}.')
+        logger.info(gen_log_message(message=msg, step=0, subject=subject,
+                                    session=session))
+        montage = mne.channels.make_standard_montage(montage_name)
+        raw.set_montage(montage, match_case=False, on_missing='warn')
+
+
+def fix_stim_artifact_func(raw: mne.io.BaseRaw) -> None:
+    """Fix stimulation artifact in the data."""
+    if not fix_stim_artifact:
+        return
+
+    events, _ = mne.events_from_annotations(raw)
+    mne.preprocessing.fix_stim_artifact(
+        raw, events=events, event_id=None,
+        tmin=stim_artifact_tmin,
+        tmax=stim_artifact_tmax,
+        mode='linear'
+    )
+
+
+def import_experimental_data(
+    *,
+    subject: str,
+    session: Optional[str] = None,
+    run: Optional[str] = None,
+    save: bool = False
+) -> mne.io.BaseRaw:
+    """Run the data import.
+
+    Parameters
+    ----------
+    subject
+        The subject to import.
+    session
+        The session to import.
+    run
+        The run to import.
+    save
+        Whether to save the data to disk or not.
+    """
+    bids_path_in = BIDSPath(subject=subject,
+                            session=session,
+                            run=run,
+                            task=get_task(),
+                            acquisition=acq,
+                            processing=proc,
+                            recording=rec,
+                            space=space,
+                            suffix=get_datatype(),
+                            datatype=get_datatype(),
+                            root=get_bids_root())
+    bids_path_out = bids_path_in.copy().update(suffix='raw',
+                                               extension='.fif',
+                                               root=get_deriv_root(),
+                                               check=False)
+
+    raw = load_data(bids_path_in)
+    set_eeg_montage(raw=raw, subject=subject, session=session)
+    create_bipolar_channels(raw=raw, subject=subject, session=session)
+    drop_channels_func(raw=raw, subject=subject, session=session)
+    rename_events_func(raw=raw, subject=subject, session=session)
+    fix_stim_artifact_func(raw=raw)
+    find_bad_channels(raw=raw, subject=subject, session=session,
+                      task=get_task(), run=run)
```

Comment:
if certain functions are only called in the config.py can you prefix them with _ as if it was private function?